packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/model/NotificationCategory.java

```diff
@@ -9,6 +9,8 @@
 
 /**
  * A class representing a collection of notification actions.
+ *
+ * TODO vonovak: no need to implement serializable, parcelable is enough for storing
```

Comment:
I had thought that categories were serialized and sent to JS as part of events, so maybe Serializable is needed here?